src/webui/www/README.md

```diff
@@ -0,0 +1,19 @@
+qBittorrent Web UI
+---
+
+### Browser compatibility
+
+| Browser           | Lower bound                                        |
+| ----------------- | -------------------------------------------------- |
+| Chrome            | [The release from 1 year ago][Chrome-history-link] |
+| Firefox           | [Oldest active ESR release][Firefox-ESR-link]      |
+| Internet Explorer | 11                                                 |
+| Microsoft Edge    | [The release from 1 year ago][MSEdge-history-link] |
+| Safari (macOS)    | [The release from 1 year ago][Safari-history-link] |
```

Comment:
If there’s ‘macOS’ here, should there also be ‘Windows’ after IE? I personally think we can drop them both.

Reply:
Removed.